src/Avalonia.Controls/TextBox.cs

```diff
@@ -30,7 +30,10 @@ public class TextBox : TemplatedControl, UndoRedoHelper<TextBox.UndoRedoState>.I
                 nameof(CaretIndex),
                 o => o.CaretIndex,
                 (o, v) => o.CaretIndex = v);
-        
+
+        public static readonly StyledProperty<char> PasswordCharProperty =
```

Comment:
Warning: OCD trigger! The rest of these property definitions are alphabetically ordered.

Reply:
Reordered :)